src/uct/tcp/tcp_cm.c

```diff
@@ -305,14 +304,13 @@ uct_tcp_cm_simult_conn_accept_remote_conn(uct_tcp_ep_t *accept_ep,
     status = uct_tcp_ep_move_ctx_cap(accept_ep, connect_ep,
                                      UCT_TCP_EP_CTX_TYPE_RX);
     if (status != UCS_OK) {
-        return status;
+        return 0;
```

Comment:
btw, looks like `uct_tcp_ep_move_ctx_cap` can be void (always returns `UCS_OK`), can update and remove this check

Reply:
`uct_tcp_ep_move_ctx_cap` can return `UCS_ERR_NO_MEMORY` in case of:
- `uct_tcp_ep_remove_ctx_cap` -> `uct_tcp_cm_add_ep` fails on memory allocation
- `uct_tcp_ep_add_ctx_cap` -> `uct_tcp_cm_add_ep` fails on memory allocation